client-app/shared/static-content/components/slider.vue

```diff
@@ -93,4 +142,15 @@ const wrapperClasses = computed(() => {
     }
   }
 }
+
+.slider__image {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  object-position: left;
+}
+
+.image-container {
```

Comment:
it's better to use inline style attr

Reply:
done

Reply:
because inside the Swiper it gives error
![image](https://github.com/VirtoCommerce/vc-theme-b2b-vue/assets/19205780/3f77a74e-e5a8-4b8c-8202-e46d48af96c2)